src/main/frontend/db/query_dsl.cljs

```diff
@@ -60,8 +61,8 @@
       (= "tomorrow" input)
       (db-utils/date->int (t/plus (t/today) (t/days 1)))
 
-      (text/page-ref? input)
-      (let [input (-> (text/page-ref-un-brackets! input)
+      (page-ref/page-ref? input)
+      (let [input (-> (page-ref/get-page-name input)
```

Comment:
Previously we were calling a `text` fn that was needlessly complex as that fn knew checked for org/markdown formats while query syntax does not care about formats. Changed query-related fns to use simpler `page-ref` fns